client/js/lounge.js

```diff
@@ -17,7 +17,6 @@ $(function() {
 		"/notice",
 		"/op",
 		"/part",
-		"/query",
```

Comment:
So it was (see `src/plugins/inputs/whois.js`, below). I'm guessing @astorije assumed, as I did at first glance, that `/query` was an alias for `/msg`. That is the common usage among other clients.